wazo_calld/plugins/calls/exceptions.py

```diff
@@ -32,6 +32,16 @@ def __init__(self, call_id):
         )
 
 
+class CallOriginUnavailableError(APIException):
+    def __init__(self, call_id):
```

Comment:
`call_id` is not the right name for what is actually the "interface name" or "interface id" or "endpoint name".
`call_id` would be the id of a created call, corresponding to a created asterisk channel.